src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -972,25 +978,29 @@ public Parent validateParent(Parent parent, String chemist, Collection<BulkLoadP
 				}
 			}
 		}
-		// check for multiple fragments, and if isMixture or not
```

Comment:
I'm confused by all the changes down in this section. As far as I can tell, you wrapped the previous logic in an `if(parent.getId() != null` block. So we're only checking pre-existing parents. Maybe if you reversed that logic it would make more sense - so we'd only be checking new parents.